### PR TITLE
Development of Navbar

### PR DESCRIPTION
**Pull request by tennjugu**
This PR closes issue  #4

### CODE REVIEW

File: dist/index.html
Comment (Bazzy09) [inline dist/index.html:1]: you are good to go

File: src/index.js
Comment (Bazzy09) [inline src/index.js:1]: good job, the code is well written for it intended purpose.

**Review by Bazzy09 — APPROVED** [review]

**Review by Bazzy09 — APPROVED** [review]

File: src/index.js
Comment (tennjugu) [reply src/index.js:1]: Thanks for the positive feedback.

File: dist/css/styles.css
Comment (Bazzy09) [inline dist/css/styles.css:1]: you did a great job in making sure the code is well structured.

**Review by Bazzy09 — APPROVED** [review]

**Pull request merged**